banner.js

```diff
@@ -6,7 +6,7 @@ const fs = require('fs');
 const pkg = require('./package.json');
 
 const filename = 'anchor.min.js';
-const padStart = str => ('0' + str).slice(-2);
+const padStart = str => String(str).padStart('0', 2);
```

Comment:
Looking at [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart), I think we might need to switch the params to `.padStart(2, '0');`

Reply:
oops yeah, typo.